src/library/qasm_ast.hpp

```diff
@@ -143,6 +143,17 @@ namespace compiler
                 type_ = toLowerCase(type);
             }
 
+            Operation(const std::string type, const int waitInt, Qubits qubits_involved)
+            : qubits_ (qubits_involved),
+              rotation_angle_ (std::numeric_limits<double>::max()), bit_controlled_(false)
+            // Single qubit wait
+            // NOTE to avoid ambigious overload with single qubit rotation (double),
+            // the int is the first argument (FIXME?)
+            {
+                type_ = toLowerCase(type);
+                wait_time_ = waitInt;
```

Comment:
```suggestion
                wait_time_ = wait_int;
```
camelCase waitInt is inconsistent here. Turn into snake_case

Reply:
Change to snake_case